src/cascadia/TerminalCore/TerminalSelection.cpp

```diff
@@ -253,16 +277,10 @@ const std::wstring Terminal::RetrieveSelectedTextFromBuffer(bool trimTrailingWhi
 // Arguments:
 // - position: viewport coordinate for selection
 // Return Value:
-// - update _selectionAnchor to new expanded location
-void Terminal::_ExpandDoubleClickSelectionLeft(const COORD position)
+// - updated copy of "position" to new expanded location (with vertical offset)
+COORD Terminal::_ExpandDoubleClickSelectionLeft(const COORD position) const
 {
-    // don't change the value if at/outside the boundary
-    if (position.X <= 0 || position.X >= _buffer->GetSize().RightInclusive())

```

Comment:
You moved this `if` block into `_ConvertToBufferCell`, but then removed the two calls to `_ConvertToBufferCell`. Was that intentional? Do these methods still need this check?

Reply:
Yup. Turns out, calling `_ConvertToBufferCell` in `_ExpandDoubleClickSelection*` was redundant. I already call it at the beginning of `DoubleClickSelection()`. So, now I just pass in the output from there.